lib/interfaces/event/index.js

```diff
@@ -43,6 +43,10 @@ export const eventTypeFields = {
     type: GraphQLString,
     description: 'Event title',
   },
+  calendarLink: {
+    type: GraphQLString,
+    description: 'Link to calendar event',
```

Comment:
Should we adjust description to "URL of the calendar event" or something? `Link` implies that there is URL _and_ text.